src/extra/CMakeLists.txt

```diff
@@ -35,7 +35,7 @@ endif()
 
 target_link_libraries(chainblocks-extra
```

Comment:
Are all those things statically linked no?
static linking destroys inlining, this is a breaking chance that needs to be profiled.

Reply:
Many of these are targets that only contain include paths just to group together what belongs to what.

cpp/c files that were previously added directly to the chainblocks target are now static libraries but I believe this doesn't affect inlining since that happens through headers only